Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: down deploy api-exec
+
+#setup image tag to latest for dev
+IMAGE_TAG?=latest
+
+# stop containers
+down:
+	docker compose down
+
+# run the deploy script
+deploy:
+	sudo ./deploy.sh
```

Comment:
Why sudo here and nowhere else?  Are you assuming users in docker group?  That's not the case on angwin cluster.
Doesn't deploy.sh now require -d/--deployment-type??